packages/drivers/odsp-driver/src/epochTracker.ts

```diff
@@ -418,6 +418,17 @@ export class EpochTracker implements IPersistedFileCache {
 export class EpochTrackerWithRedemption extends EpochTracker {
     private readonly treesLatestDeferral = new Deferred<void>();
 
+    constructor(
+        protected readonly cache: IPersistedCache,
+        protected readonly fileEntry: IFileEntry,
+        protected readonly logger: ITelemetryLogger,
+        protected readonly clientIsSummarizer?: boolean,
+    ) {
+        super(cache, fileEntry, logger, clientIsSummarizer);
+        // Handles the rejected promise within treesLatestDeferral.
```

Comment:
Can you say more about why this is necessary?  From my recollection the reason is that we cause this promise to reject when a particular request fails, and that failure is propagated just fine.  But we only await the promise under a certain set of conditions, maybe a certain sequence of requests.  So during the interim time between requests, the rejection is truly unhandled.

Reply:
Yeah so, the actual response to join session/trees latest request or other calls are truly returned. This promise was more of waiting logic in case join session call is made before the trees latest call. This promise is imp only in this class and its rejection does not need to be propagated. So, we just need to add a empty catch statement so as to avoid unhandledPromiseRejection error. Also adding this description above.